### PR TITLE
refactor(dropdown): tailwindify 

### PR DESCRIPTION
**Pull request by julio8a**
Supports #1500 

Width change on this one to support scale
s - 48
m - 56
l - 64

^ from spacing scale

### CODE REVIEW

File: src/components/calcite-combobox/calcite-combobox.scss [hunk 1]
```diff
@@ -137,21 +138,18 @@
 }
 
 .list-container {
-  overflow-y: auto;
-  max-height: 100vh;
+  @apply overflow-y-auto
+    max-h-screen;
   width: var(--calcite-dropdown-width);
   background: var(--calcite-ui-foreground-1);
```
Comment (paulcpederson) [inline src/components/calcite-combobox/calcite-combobox.scss:144]: `bg-foreground-1` I think

Comment (julio8a) [reply src/components/calcite-combobox/calcite-combobox.scss]: I accidentally have Combobox in this pr when it's already here: https://github.com/Esri/calcite-components/pull/1789

That said, I did update this in the Combobox pr. thx dude!

**Comment by julio8a** [comment]
anything else on this one?

File: src/components/calcite-dropdown/calcite-dropdown.scss [hunk 2]
```diff
@@ -1,13 +1,10 @@
 :host {
-  position: relative;
-  display: inline-flex;
-  flex-grow: 1;
+  @apply relative inline-block flex-grow;
```
Comment (jrowlingson) [inline src/components/calcite-dropdown/calcite-dropdown.scss:2]: Assume inline-block was intended.

**Comment by julio8a** [comment]
Ready for review

**Review by jrowlingson — APPROVED** [review]

**Comment by julio8a** [comment]
adding dropdown-group and dropdown-item here

**Comment by julio8a** [comment]
This one got a little messy, going to open it as a new PR.

**Pull request closed without merging**